P1072.bs

```diff
@@ -768,36 +807,44 @@ constexpr void resize_and_overwrite(size_type n, Operation op);
 <li><ins>— `k` be `min(o, n)`.</ins></li>
 
 <li><ins>— `p` be a `charT*`, such that the range `[p, p + n]` is
-valid, and `string_view(p, k) == *this` before the call. The values in
-the range `[p + k, p + n]` may be indeterminate
+valid and `this->compare(0, k, p, k) == 0` is `true` before the call.
+The values in the range `[p + k, p + n]` may be indeterminate
 [**basic.indet**].</ins></li>
 
-<li><ins>— `size_type r = op(p, n)`.</ins></li>
+<li><ins>— `r` = `op(p, n)`.</ins></li>
 
 </ul>
 
+<li><ins>*Mandates:* `op(p, n)` has an integer-like type
+([**iterator.concept.winc**]).</ins></li>
+
 <li><ins>*Preconditions:*</ins></li>
 
 <ul style="list-style-type:none;">
 
-<li><ins>— `op` does not throw an exception.</ins></li>
+<li><ins>— `op(p, n)` does not throw an exception.</ins></li>
+
+<li><ins>— `r` &#x2265 `0`.</ins></li>
```

Comment:
Nit: Does `&gte;` and related abbreviations not work here?  `x2265` is a bit subtle...